src/spoke/interfaces/ISpoke.sol

```diff
@@ -18,28 +19,30 @@ interface ISpoke is ISpokeBase, IMulticall, INoncesKeyed, IAccessManaged {
   /// @dev assetId The identifier of the asset in the Hub.
   /// @dev decimals The number of decimals of the underlying asset.
   /// @dev dynamicConfigKey The key of the last reserve dynamic config.
-  /// @dev paused True if all actions are prevented for the reserve.
-  /// @dev frozen True if new activity is prevented for the reserve.
-  /// @dev borrowable True if the reserve is borrowable.
   /// @dev collateralRisk The risk associated with a collateral asset, expressed in BPS.
+  /// @dev flags The packed boolean flags of the reserve (a wrapped uint8).
   struct Reserve {
     address underlying;
     //
     IHubBase hub;
     uint16 assetId;
     uint8 decimals;
     uint24 dynamicConfigKey;
-    bool paused;
-    bool frozen;
-    bool borrowable;
     uint24 collateralRisk;
+    ReserveFlags flags;
   }
 
   /// @notice Reserve configuration. Subset of the `Reserve` struct.
+  /// @dev paused True if all actions are prevented for the reserve.
+  /// @dev frozen True if new activity is prevented for the reserve.
+  /// @dev borrowable True if the reserve is borrowable.
+  /// @dev receiveSharesEnabled True if the liquidator can receive collateral shares during liquidation.
+  /// @dev collateralRisk The risk associated with a collateral asset, expressed in BPS.
   struct ReserveConfig {
     bool paused;
     bool frozen;
     bool borrowable;
+    bool receiveSharesEnabled;
     uint24 collateralRisk;
```

Comment:
should we put this one first just in case we add more flags later, they don't end up in the middle of the struct?